frontend/packages/design-system/src/themes/index.ts

```diff
@@ -9,4 +9,13 @@ export default extendTheme({
   components: {
     Button,
   },
+  space: {
+    '1': tokens.spacing['spacing-5'],
```

Comment:
I have a small preference for `spacing5` so that we can do `tokens.spacing.spacing5`